### PR TITLE
[WIP] Add scholarship form / adjust text

### PR DESCRIPTION
**Pull request by verpixelt**

### CODE REVIEW

File: scholarships.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
 layout: subpage
 page_id: scholarships
 title: "Apply for a scholarship"
-subtitle: "Applications open soon"
```
Comment (Kriesse) [inline scholarships.html:5]: Maybe change this to:
"Applications open until January 20" (or whatever date we decide)

File: scholarships.html [hunk 2]
```diff
@@ -29,5 +28,5 @@
 </p>
 
 <h2>Application Form</h2>
-
-<p>Applications will open soon. <a href="https://twitter.com/cssconfeu">Follow us on Twitter</a> and <a href="https://confirmsubscription.com/h/d/879A481DB04CB70D">subscribe to our newsletter</a> to stay up to date.</p>
+<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSfMxkhodtfYjJLJAV_q70bmcb7tiF7ubEuFRCOF_mkfLfPUMA/viewform?embedded=true" width="100%" height="1000" style="margin-bottom: 1.7143rem; "frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+<p>If you have any questions please don't hesitate to contact us. <a href="https://twitter.com/cssconfeu">Follow us on Twitter</a> and <a href="https://confirmsubscription.com/h/d/879A481DB04CB70D">subscribe to our newsletter</a> to stay up to date.</p>
```
Comment (Kriesse) [inline scholarships.html]: add email address:
`to contact us at contact@cssconf.eu`

**Review by Kriesse — CHANGES_REQUESTED** [review]
LGTM except some wording suggestions

**Review by Kriesse — APPROVED** [review]

**Comment by Kriesse** [comment]
Approved to merge, waiting on https://github.com/jsconf/2017.jsconf.eu/pull/3

**Review by lukaszklis — APPROVED** [review]

**Pull request merged**